ocp_resources/virtual_machine_clone.py

```diff
@@ -0,0 +1,69 @@
+# TODO: add a link to the resource API
```

Comment:
Found this one https://kubevirt.io/user-guide/operations/clone_api/

Reply:
oh, I was thinking we should use references from ocp doc only, added link to the kubevirt